actions/class.Tests.php

```diff
@@ -167,7 +168,14 @@ public function delete()
             $lockManager->releaseLock($instance, $userId);
         }
 
-        if ($this->service->deleteTest($instance)) {
+        // If it is a class deletion
+        if ($uri === $classUri) {
+            $class = $this->getClass($classUri);
+            $label = $class->getLabel();
+
+            $success = $this->getClassService()->deleteClass($class);
```

Comment:
Will this delete tests under the class?

Reply:
Yep 🙂

Reply:
Do you know why we have the method `taoTests_models_classes_TestsService::deleteTestClass` in the service and we do not use it? I saw it is pretty simpler than the `deleteClass` (seems wrong too). 

When 
Do you call `$this->getClassService()->deleteTest($instance);`, I saw you delete also tests file content, but for test class, It seems we do not.

Reply:
@gabrielfs7 AFAIK `taoTests_models_classes_TestsService::deleteTestClass` has no extra logic on top of what `\oat\tao\model\ClassServiceTrait::deleteClass` provides, may be because of some historical reasons. What would you suggest to change here? 

Also, as `ClassServiceTrait::deleteClass` is using `::deleteResource`, which is overridden by `taoTests_models_classes_TestsService` implementation and therefor calling `taoTests_models_classes_TestsService::deleteTestClass` I believe we are safe with content deletion even at class level operations